Builds/CMake/CMakeFuncs.cmake

```diff
@@ -35,6 +35,7 @@ macro(parse_target)
       string(REGEX REPLACE "^\\.?[^\\.]+(.*$)" "\\1" remaining ${remaining})
 
       if (${cur_component} STREQUAL gcc)
+        set(expect_cc true)
```

Comment:
If we could check `CMAKE_C_COMPILER` and `CMAKE_CXX_COMPILER` for empty strings (or `-NOTFOUND`) before setting them in the cache, I'd rather do that. If that won't work then fine as is.